web/gds-admin-ui/src/pages/account/Login.js

```diff
@@ -1,69 +1,38 @@
 // @flow
 import React from 'react';
-import { Button } from 'react-bootstrap';
-import { Link, useHistory } from 'react-router-dom';
-import * as yup from 'yup';
-import { yupResolver } from '@hookform/resolvers/yup';
-
-import { VerticalForm, FormInput } from '../../components/';
+import { Redirect } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
 
 import AccountLayout from './AccountLayout';
+import SignWithGoogle from '../../components/Auth/SignWithGoogle';
+import { loginUser } from '../../redux/auth/actions';
 
 
 const Login = (): React$Element<any> => {
-    const history = useHistory();
-
-    const onSubmit = (formData) => {
-        console.log("[Login] Form Date", formData)
-        history.push("/dashboard")
-    };
-
-    const schemaResolver = yupResolver(
-        yup.object().shape({
-            username: yup.string().required('Please enter Username'),
-            password: yup.string().required('Please enter Password')
-        })
-    );
+    const clientId = process.env.REACT_APP_GOOGLE_CLIENT_ID
```

Comment:
Perhaps it would be better to store this in config.js?

Reply:
You’re right